src/redux/reducers/index.ts

```diff
@@ -1,13 +1,29 @@
 // import all reducers
 import { combineReducers } from 'redux';
-import userStatsAPI from './userStatsAPI'
-import searchResultsAPI from './searchResultsAPI'
-import addSongToQueueAPI from './addSongToQueueAPI'
+import userStatsAPI from './userStatsAPI';
+import searchResultsAPI from './searchResultsAPI';
+import addSongToQueueAPI from './addSongToQueueAPI';
+import setAccessToken from './setAccessToken';
+import loginReducer from './loginReducer';
+import setCurrentSong from './setCurrentSong';
+import setSongPosition from './setSongPosition';
+import isLockedReducer from './isLockedReducer';
+import setPlay from './setPlay';
+import setDeviceId from './setDeviceId';
+
+
 
 const allReducers = combineReducers({
-  userStatsReducer : userStatsAPI,
-  searchResultsReducer : searchResultsAPI,
-  addSongToQueueReducer : addSongToQueueAPI
+  userStatsReducer: userStatsAPI,
+  searchResultsReducer: searchResultsAPI,
+  addSongToQueueReducer: addSongToQueueAPI,
+  access_token: setAccessToken,
+  isLoggedIn: loginReducer,
+  current_song: setCurrentSong,
+  position: setSongPosition,
+  isLocked: isLockedReducer,
+  playing: setPlay,
+  device_id : setDeviceId
```

Comment:
Requirements: It seems like we need some new properties in the state.

Me: Let's add them here 🥳
![](https://media.giphy.com/media/2p15M2c5lmc7u/giphy.gif)